sys-libs/libxcrypt/libxcrypt-4.4.19.ebuild

```diff
@@ -52,19 +61,50 @@ src_configure() {
 	multibuild_foreach_variant multilib-minimal_src_configure
 }
 
+get_xcsysroot() {
+	if is_cross; then
+		echo "${EPREFIX}/usr/${CTARGET}"
+	else
+		echo "/"
+	fi
+}
+
 get_xclibdir() {
-	printf -- "%s/%s/%s\n" \
+	printf -- "%s/%s/%s/%s\n" \
+		"$(get_xcsysroot)" \
 		"$(usex split-usr '' '/usr')" \
 		"$(get_libdir)" \
 		"$(usex system '' 'xcrypt')"
 }
 
+get_xcincludedir() {
+	printf -- "%s/%s/include/%s\n" \
+		"$(get_xcsysroot)" \
+		"$(usex split-usr '' '/usr')" \
+		"$(usex system '' 'xcrypt')"
+}
+
+get_xcmandir() {
+	printf -- "%s/%s/share/man\n" \
+		"$(get_xcsysroot)" \
```

Comment:
sysroot is wrong for this; you want ${EPREFIX}/usr/${CTARGET} or ${EPREFIX} depending on cross or not, because this is for the final product - not where we find dependencies.